server/test/user.test.js

```diff
@@ -0,0 +1,121 @@
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const { server } = require("../app.js");
+
+chai.should();
+chai.use(chaiHttp);
+
+const register_detailUsername={
```

Comment:
camelCase is the standard naming convention for variables in JS, please change accordingly

Ditto throughout

Reply:
Done changes